blosc/blosc2.c

```diff
@@ -3348,7 +3364,13 @@ int blosc_get_complib_info(const char* compname, char** complib, char** version)
   }
 #if defined(HAVE_ZLIB)
   else if (clibcode == BLOSC_ZLIB_LIB) {
+#ifdef ZLIB_COMPAT
     clibversion = ZLIB_VERSION;
+#elif defined(HAVE_ZLIB_NG)
+    clibversion = ZLIBNG_VERSION;
```

Comment:
Does the return value of clib version get saved in the file? 

Zlibng version will be in the 2.x.x while zlib version. 1.x.x which may be an important change

Reply:
And potentially b undesirable

Reply:
That's a good question.  Maybe @nmoinvaz can shed some light here.

Reply:
Ups, sorry, `clibversion` is only used for `blosc_get_complib_info()` to get the versions of internal codecs, so not stored in the file.  With that, I think we are safe merging that.